lib/test_queue/runner.rb

```diff
@@ -150,7 +150,7 @@ def summarize_internal
           worker.suites.size,
           worker.end_time - worker.start_time,
           worker.pid,
-          worker.status.exitstatus,
+          worker.status.exitstatus || 1,
```

Comment:
It'd be nice to show the signal name if the process was killed by a signal. In fact, it looks like we could replace the whole `pid %d exit %d` portion of the string with `worker.status.to_s`, which gives us that same information including info about the signal.